src/Aspire.Cli/Program.cs

```diff
@@ -268,6 +278,35 @@ await AnsiConsole.Live(table).StartAsync(async context => {
         parentCommand.Subcommands.Add(command);
     }
 
+    private static async Task EnsureCertificatesTrustedAsync(DotNetCliRunner runner, CancellationToken cancellationToken)
+    {
+        var checkExitCode = await AnsiConsole.Status()
+            .Spinner(Spinner.Known.Dots3)
+            .SpinnerStyle(Style.Parse("purple"))
+            .StartAsync(
+                ":locked_with_key: Checking certificates...",
+                async (context) => {
+                    return await runner.CheckHttpCertificateAsync(cancellationToken);
```

Comment:
This will only check if the certificate is installed, not if it's trusted. Is that intended? You need to pass both `--check` and `--trust` to check the trust status of installed certificates.

Reply:
Fixed.